gemonlinedb/include/gem/onlinedb/VFAT3ChannelConfiguration.h

```diff
@@ -0,0 +1,133 @@
+#ifndef _VFAT3ChannelConfiguration_h_
+#define _VFAT3ChannelConfiguration_h_
+
+#include <array>
+#include <cstdint>
+
+#include "gem/onlinedb/ConfigurationTraits.h"
+#include "gem/onlinedb/PartReference.h"
+#include "gem/onlinedb/detail/RegisterData.h"
+
+namespace gem {
+    namespace onlinedb {
+        /**
+         * @brief Describes the configuration of channels in a VFAT 3
+         */
+        class VFAT3ChannelConfiguration
+        {
+        public:
+            /**
+             * @brief The number of channels in a VFAT 3
+             */
+            static const constexpr std::size_t channelCount = 128;
+
+            /**
+             * @brief Represents the value of a channel register.
+             */
+            union ChannelRegisterValue
+            {
+                /**
+                 * @brief The raw value.
+                 */
+                std::uint16_t registerValue;
+
+                /**
+                 * @brief Individual settings making up the value.
+                 */
+                struct {
+                    /**
+                     * @brief Arming comparator threshold trimming: amplitude.
+                     */
+                    std::uint16_t armDacCorrectionAmplitude : 6;
```

Comment:
Mentioned to @bdorney in a different PR, adopt an "almost" camel case style, i.e., a DAC is a thing (not a `Dac` or a `dac`), so if it's the second word in a variable name, it should be all caps, if it's the first word, all lower case; for words that don't have a specific capitalization, standard camel case should be used